src/mmw/js/src/modeling/models.js

```diff
@@ -15,9 +15,75 @@ var ResultCollection = Backbone.Collection.extend({
     model: ResultModel
 });
 
-var ProjectModel = Backbone.Model.extend({});
+var ProjectModel = Backbone.Model.extend({
+    urlRoot: '/api/modeling/projects/',
+
+    initialize: function() {
+        // TODO: For a new project, users will eventually
+        // be able to choose which modeling package
+        // they want to use in their project. For
+        // now, the only option is TR55, so it is
+        // hard-coded here. This will likely be a model
+        // itself in the future, with more modification
+        // and input control data.
+        this.set('model_package', 'tr-55');
+        this.set('active_scenario_slug', 'current-conditions');
+        this.set('taskModel', new Tr55TaskModel());
+    },
+
+    saveAll: function() {
+        var self = this;
+
+        // If we haven't saved the project before,
+        // hold on to the scenarios so that we can
+        // save each one once we have an ID for the project
+        if (this.isNew) {
+            var tempScenarios = this.get('scenarios');
+        }
```

Comment:
Before we can save the scenarios, we have to save the project so that we know the ID of the project. If we save the project without temporarily storing the scenarios, we'll lose them when we refresh our project model from the server response because the scenarios are independent models on the server and not yet saved. Once we have the project ID, we can iterate through the scenarios and save them.